dspace-api/src/main/java/org/dspace/core/Context.java

```diff
@@ -571,32 +591,32 @@ public boolean isReadOnly() {
         return mode != null && mode == Mode.READ_ONLY;
     }
 
+    /**
+     * Add a group's UUID to the list of special groups cached in Context
+     * @param groupID UUID of group
+     */
     public void setSpecialGroup(UUID groupID) {
         specialGroups.add(groupID);
-
-        // System.out.println("Added " + groupID);
     }
 
     /**
-     * test if member of special group
+     * Test if a group is a special group
      *
      * @param groupID ID of special group to test
      * @return true if member
      */
     public boolean inSpecialGroup(UUID groupID) {
         if (specialGroups.contains(groupID)) {
-            // System.out.println("Contains " + groupID);
```

Comment:
Could be logged at DEBUG level.  Not a big deal.

Reply:
I'm going to just leave this as removed. I agree it could have been solved differently via `log.debug`. But, as it was already commented out for some time, I don't know how useful it is anymore.  If someone finds a use for it, it's easy to add in later.